frontend/src/views/Collection.vue

```diff
@@ -0,0 +1,254 @@
+            </dl>
+          </section>
+        </section>
+        <section id="sitemap" v-if="sites.length > 0">
+          <header>Sites</header>
+          <section class="details">
+            <Map
+                :sites="sites"
+                :center="[54.00, 14.00]"
+                :zoom="3"
+            ></Map>
+          </section>
+        </section>
+        <section id="products">
+          <header>Products</header>
+          <section class="details">
+            <div v-for="product in products" :key="product.id">
+              <img :src="getIconUrl(product.id)" class="product">
+              {{ product.humanReadableName }}
+            </div>
+          </section>
+        </section>
+      </main>
+      <div class="rightView">
+        <nav class="internalNavi">
+          <router-link :to="{path: `general`}" :replace="true" v-bind:class="{ 'router-link-active': mode === 'general' }">
+            General
+          </router-link> |
+          <router-link :to="{path: `files`}" :replace="true" v-bind:class="{ 'router-link-active': mode === 'files' }">
+            All files
+          </router-link>
+        </nav>
+        <section id="editCollection" class="rightView" v-if="mode === 'general'">
+          <h3>Citing</h3>
+          <span v-if="busy">Generating PID...</span>
+          <a v-if="response.pid" :href="response.pid">{{response.pid }}</a>
+          <div v-if="pidServiceError" class="errormsg">PID service is unavailable. Please try again later.</div>
+          <h3>License</h3>
+          <!-- eslint-disable max-len -->
+          Cloudnet data is licensed under a <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 international licence</a>.
+          <p>
+            This is a human-readable summary of (and not a substitute for) the <a href="https://creativecommons.org/licenses/by/4.0/legalcode">licence</a>.<br>
+            You are free to:
+          <ul>
+            <li><b>Share</b> — copy and redistribute the material in any medium or format</li>
+            <li><b>Adapt</b> — remix, transform, and build upon the material for any purpose, even commercially</li>
+          </ul>
+          Under the following terms:
+          <ul>
+            <li><b>Attribution</b> — You must give appropriate credit, provide a link to the licence, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use</li>
+            <li><b>No additional restrictions</b> — You may not apply legal terms or technological measures that legally restrict others from doing anything the licence permits</li>
+          </ul>
+The licensor cannot revoke these freedoms as long as you follow the licence terms.
+          <!-- eslint-enable max-len -->
+          <h3>Download</h3>
+          By clicking the download button you confirm that you have taken notice of the above data licensing information.<br>
```

Comment:
Sano napakammin jos pystyt, jotenkin "By clicking the download button you agree to follow these terms." tms.